README.rst

```diff
@@ -655,7 +655,8 @@ without filtering by the organization the user has access to, which
 breaks multi-tenancy.
 
 The ``FilterSerializerByOrgMembership``, ``FilterSerializerByOrgManaged``
-and ``FilterSerializerByOrgOwned`` can be used to solve this issue.
+and ``FilterSerializerByOrgOwned`` can be used to solve this issue. These
+serializers obstructs non super-users from creating shared objects.
```

Comment:
Please move the new sentence on its own line and let's change it to:

"These serializers do not allow non-superusers to create shared objects".